source/agora/consensus/EnrollmentManager.d

```diff
@@ -1236,7 +1200,7 @@ unittest
     assert(man.addValidator(ordered_enrollments[2], WK.Keys[2].address, Height(1019),
             &utxo_set.peekUTXO, utxos) is null);
     man.clearExpiredValidators(Height(1019));
-    assert(man.countActiveIfExternalized(Height(1019)) == 1);
+    assert(man.validator_set.countActive(Height(1019)) == 1);
```

Comment:
Actually these are correct. The test just passed before because it is also true for next block. I added some tests at end of cycle boundary to fail should the wrong height be used in the future.